custom/active-speaker/components/SpeakerView/SpeakerView.js

```diff
@@ -3,67 +3,41 @@ import { Container } from '@custom/basic-call/components/Call/Container';
 import Header from '@custom/basic-call/components/Call/Header';
 import ParticipantBar from '@custom/shared/components/ParticipantBar/ParticipantBar';
 import VideoContainer from '@custom/shared/components/VideoContainer/VideoContainer';
-import { useCallState } from '@custom/shared/contexts/CallProvider';
 import { useParticipants } from '@custom/shared/contexts/ParticipantsProvider';
 import { useTracks } from '@custom/shared/contexts/TracksProvider';
-import { useUIState } from '@custom/shared/contexts/UIStateProvider';
-import { isScreenId } from '@custom/shared/contexts/participantsState';
+import { Screens } from './Screens';
 import { SpeakerTile } from './SpeakerTile';
 
 const SIDEBAR_WIDTH = 186;
 
 export const SpeakerView = () => {
-  const { currentSpeaker, localParticipant, participants, screens } =
+  const { currentSpeakerId, localParticipant, screens, orderedParticipantIds } =
```

Comment:
To me it's confusing to use `localParticipant` from the `useParticipants` provider still. Why not the Daily Hook? Is there a use case where we would need it from the participants provider? If not, it can be removed